storage/innobase/xtrabackup/src/xbcloud.cc

```diff
@@ -602,7 +602,8 @@ swift_temp_auth(const char *auth_url, swift_auth_info *info)
 			goto cleanup;
 		}
 		curl_easy_getinfo(curl, CURLINFO_RESPONSE_CODE, &http_code);
-		if (http_code != 200) {
+		if (http_code != 200 &&
+			http_code != 204) {
```

Comment:
Please change indentation to align second `http_code` with the first one. See the line 772 for example.